tests/unit/test-util/test.manifest.js

```
@@ -60,23 +61,30 @@ describe('util/manifest', () => {
         }));
     });
 
-    it('reports an error for invalid manifest JSON', () => withTempDir(
-      (tmpDir) => {
-        const badManifest = `{
-          "name": "I'm an invalid JSON Manifest
-          "version": "0.0.0"
-        }`;
-        const manifestFile = path.join(tmpDir.path(), 'manifest.json');
-        return fs.writeFile(manifestFile, badManifest)
-          .then(() => getValidatedManifest(tmpDir.path()))
-          .then(makeSureItFails())
-          .catch(onlyInstancesOf(InvalidManifest, (error) => {
-            assert.match(error.message, /Error parsing manifest\.json at /);
-            assert.include(error.message, 'Unexpected token \' \' at 2:49');
-            assert.include(error.message, manifestFile);
-          }));
+    // FIXME: Failed on Node 7.
+    // https://github.com/sindresorhus/parse-json/issues/6
+    it('reports an error for invalid manifest JSON', function() {
+      if (semver.gte(process.versions.node, '7.0.0')) {
```

Again, I think we should let it fail for Node 7 for now